bin/master_npid_updater.rb

```diff
@@ -0,0 +1,90 @@
+require 'zlib'
+
+config   = Rails.configuration.database_configuration
+@username = config[Rails.env]["username"]
+@password = config[Rails.env]["password"]
+@database_main = config[Rails.env]["database"]
+@batch_size = 1_000
+
+
+def init_insert
+	sql = "INSERT INTO `location_npids` (`couchdb_location_npid_id`, `couchdb_location_id`, `location_id`, `npid`, `assigned`,`created_at`, `updated_at`) VALUES "
+end
+
+def load_file(db, file)  
+  puts "loading #{file} into #{db}"
+  infile = open(file)
+  gz = Zlib::GzipReader.new(infile)
+
+  statements = gz.split(/;$/)
+  statements.pop

```

Comment:
To handle cases when there are newlines after the last ';'? Won't you drop the last query if there isn't a new line after the last query?

```ruby

2.5.8 :022 > "foo;\nbar;\nfoobar;".split(/;$/)
 => ["foo", "\nbar", "\nfoobar"]

```

Reply:
We are not using this function I decided not to use it. Might do so in future but I think it will not be necessary.